pleskdistup/common/src/dpkg.py

```diff
@@ -184,13 +242,14 @@ def depconfig_parameter_get(parameter: str) -> str:
     return process.stdout.split(" ")[1].strip()
 
 
-def restore_installation() -> None:
-    util.logged_check_call(["/usr/bin/apt-get", "-f", "install", "-y"])
+def restore_installation(tmpfail_retry_intervals: typing.Optional[typing.List[int]] = None) -> None:
+    cmd = ["/usr/bin/apt-get", "-f", "install", "-y"]
+    apt_get_retry_temp_fails(cmd, tmpfail_retry_intervals)
 
 
-def do_distupgrade() -> None:
-    util.logged_check_call(["apt-get", "dist-upgrade", "-y"] + APT_CHOOSE_OLD_FILES_OPTIONS,
-                           env={"PATH": os.environ["PATH"], "DEBIAN_FRONTEND": "noninteractive"})
+def do_distupgrade(tmpfail_retry_intervals: typing.Optional[typing.List[int]] = None) -> None:
```

Comment:
Do we really need to expose `tmpfail_retry_intervals` everywhere? What for?

Reply:
I introduced `dpkg.DPKG_TEMPFAIL_RETRY` with default values - one can set this variable to influence retry behavior of all those funcs. But they can also send arguments with custom retry intervals. Depends on usability, if users of this code won't need it, then we can only depend on global variable to set the retry behavior.

Reply:
OK, let it be as is, although I don't condone setting global constants from elsewhere.